File: problems/455-assign-cookies/assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        count = 0
        gIndex = 0
        sIndex = 0
        s.sort()
        g.sort()
        while gIndex < len(g) and sIndex < len(s):
            if s[sIndex] >= g[gIndex]:
                count += 1
                gIndex += 1
                sIndex += 1
            else:
                sIndex += 1
        
        return count

File: problems/455-assign-cookies/assign-cookies.php
class Solution {

    /**
     * @param Integer[] $g
     * @param Integer[] $s
     * @return Integer
     */
    function findContentChildren($g, $s) {
        $count = 0;
        $gIndex = 0;
        $sIndex = 0;
        sort($g);
        sort($s);
        while ($gIndex < count($g) && $sIndex < count($s)) {
            if ($s[$sIndex] >= $g[$gIndex]) {
                $count ++;
                $gIndex ++;
            }
            $sIndex ++;
        }
        return $count;
    }
}